src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
+  module.exports = 'http-auth-interceptor';
```

Comment:
For me it's to use it with webpack like described here:

http://angular-tips.com/blog/2015/06/using-angular-1-dot-x-with-es6-and-webpack/

    import httpAuth from 'http-auth-interceptor';
    angular.module('app', [httpAuth]);

Basically the same thing angular itself does with its `index.js` file distributed in their npm package.